borderless/display_override-borderless-explainer.md

```diff
@@ -0,0 +1,512 @@
+# Borderless Explainer
```

Comment:
Is there a reason you named this `display_override-borderless-explainer.md`? Can you make it `borderless-explainer.md` instead?

Reply:
No reason, actually we can even just call it `explainer.md` as it's already in the borderless folder?

Reply:
All the other explainers are in the top level and called `<feature-name>-borderless.md`. I think it's probably time to move these into an `explainers` subdir but for now, can you just put it there?